### PR TITLE
Add quiz telemetry

### PR DESCRIPTION
**Pull request by ashton314**
Gets telemetry from quizzes (just a little bit of JSON) and appends it to the end of a file on the server.

This is a *rough draft* and *edits are needed*. Just looking to get some feedback on where to go from here.

### CODE REVIEW

**Comment by ashton314** [comment]
Note: this has all the commits from adding quizzes, so **DO NOT MERGE** until #1260 is merged.

File: infra/api.py
Comment (bennn) [inline infra/api.py:4]: why not `bottle.response` below? is this import even used?

Comment (ashton314) [reply infra/api.py]: Fixed it; the language server snuck this import in when I was doing local testing. I've removed it.

**Comment by pavpanchekha** [comment]
We should update the branch and mark it ready for review so Chris can look at it, but this seems like a super safe and easy PR.

**Comment by pavpanchekha** [comment]
Do you know why it still has a diff for everything? Do you need to rebase & force-push or something?

**Comment by ashton314** [comment]
Ach, yes. I'll fix that. I spun this branch off of my other dev branch and they got out of sync.

**Comment by ashton314** [comment]
Ugh—that was a messy rebase. Smaller diff with main now.

I hid the quiz bug report button; I figure if there's an issue, people can submit a comment via the usual mechanism. If you *want* that separate bug reporting mechanism, I can figure out how to enable that in a separate PR.

**Review by pavpanchekha — APPROVED** [review]

File: infra/api.py
Comment (pavpanchekha) [inline infra/api.py:13]: Maybe end it with `.json`?

File: infra/api.py
Comment (pavpanchekha) [inline infra/api.py:126]: Do quiz questions have some kind of stable ID? If we change the questions how do we track 1) which version the answers are for, and 2) what question it was?

File: infra/api.py
Comment (ashton314) [reply infra/api.py:13]: It'll be a bunch of JSON objects, not suitable for parsing as-is. You'd have to wrap brackets around it. IDK if calling that `.json` will mess up any processing tools. I can call it that. Any other changes to that file name/path while I'm changing that?

File: infra/api.py
Comment (ashton314) [reply infra/api.py:126]: I don't think there's a versioning system—that's a really good idea though. I can see what I can put together this week.

File: infra/api.py
Comment (pavpanchekha) [reply infra/api.py:13]: I think the file name is fine. JSON is sort of a disaster of a format, but I guess we can use `.json` anyway. As you can see earlier in the file, we use a single-object `.json` for some other data, but I agree that loading & saving the whole object isn't a good idea, it'll grow to eventually be super slow.

File: www/quiz_style.css
Comment (chrishtr) [inline www/quiz_style.css:155]: If these are going to get commented out, just remove them in this PR

Comment (ashton314) [reply www/quiz_style.css]: Done; I took out all the `.bug-report` styles

**Pull request merged**
